### DISCUSSION TITLE
ScriptReference/Time-realtimeSinceStartup

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#231] [Notes]
# ScriptReference/Time-realtimeSinceStartup

https://docs.unity3d.com/ScriptReference/Time-realtimeSinceStartup.html

**Comment by mvi** [comment]
[Here](https://forum.unity.com/threads/unscaledtime-vs-realtimesincestartup.533310/#post-3512137) is an excellent explanation of the difference between `Time.realtimeSinceStartup` and `Time.unscaledTime`.
